tutorials/misc/gles2_gles3_differences.rst

```diff
@@ -0,0 +1,79 @@
+.. _doc_gles2_gles3_differences:
+
+Differences between GLES2 and GLES3
+===================================
+
+This page documents the differences between GLES2 and GLES3 that are by design and are not the result
+of bugs. There may be differences that are unintentional, but they should be reported as bugs.
+
+Particles
+---------
+
+GLES2 cannot use the :ref:`Particles <class_Particles>` or :ref:`Particles2D <class_Particles2D>` nodes
+as they require advanced GPU features. Instead use CPUParticles, which provides a similar interface to
+a :ref:`ParticlesMaterial <class_ParticlesMaterial>`.
+
+SCREEN_TEXTURE mip-maps
+-----------------------
+
+In GLES2 ``SCREEN_TEXTURE`` (accessed via a :ref:`ShaderMaterial <class_ShaderMaterial>`) does not have
+computed mip-maps. So when accessing at a different LOD, the texture will not appear blurry. 
+
+Color space
+-----------
+
+Shading in GLES3 takes place in linear color space while in GLES2 it takes place in sRGB space.
+While this is a very important distinction for graphics programmers, for everyone else it means
+that in GLES3 shaders you need to convert to sRGB before outputting your final color or else
+your colors will appear washed out and over-bright. 
+
+You do so with the following line of code at the end of the fragment shader:
+
+.. code-block:: glsl
+
+		ALBEDO =  mix(pow((ALBEDO + vec3(0.055)) * (1.0 / (1.0 + 0.055)),vec3(2.4)),ALBEDO * (1.0 / 12.92),lessThan(ALBEDO,vec3(0.04045)));
+
+GLES2 does not need this. If your game is going to use both GLES2 and GLES3, wrap it in an ``if`` 
+statement checking to see if the output is in sRGB, using ``OUTPUT_IS_SRGB``. 
+
```

Comment:
I think this may be a bit misleading since in most cases it does not really matter and user will be ok with this looking a bit different. Also using mix and  lessThan is very un-optimal in OpenGL ES 2.0 because it's emulated in code (mix does not really accept booleans in GLES2.

I would remove entirely the converting comment and just specify that they are in different colorspaces, which will result in them looking a bit different when light is applied (which is all that really matters).. without proposing anything else.

Reply:
This was added as a direct response to [this issue](https://github.com/godotengine/godot/issues/25142). The issue is that they look different even when no light is applied. How about I remove some of the description that could be confusing/misleading. And then remove the color correction code, but still note the use of ``OUTPUT_IS_SRGB``?